documentation/asciidoc/accessories/camera/libcamera_apps_intro.adoc

```diff
@@ -0,0 +1,34 @@
+== `libcamera` and `libcamera-apps`
+
+=== Introduction
+
+`libcamera` is a new software library aimed at supporting complex camera systems directly from the Linux operating system. In the case of the Raspberry Pi it enables us to drive the camera system directly from open source code running on ARM processors. The proprietary code running on the Broadcom GPU, and to which users have no access at all, is almost completely by-passed.
+
+`libcamera` presents a C++ API to applications and works at the level of configuring the camera and then allowing an application to request image frames. These image buffers reside in system memory and can be passed directly to still image encoders (such as JPEG) or to video encoders (such as h.264), though such ancilliary functions as encoding images or displaying them are strictly beyond the purview of `libcamera` itself.
+
+For this reason Raspberry Pi supplies a small set of example `libcamera-apps`. These are simple applications, built on top of `libcamera`, and are designed largely to emulate the function of the legacy stack built on Broadcom's propretary GPU code (some users will recognise these legacy applications as `raspstill` and `raspivid`). The applications we provide are:
+
+* _libcamera-hello_ A simple "hello world" application which starts a camera preview stream and displays it on the screen.
+* _libcamera-jpeg_ A simple application to run a preview window and then capture high resolution still images.
+* _libcamera-still_ A more complex still image capture application which emulates more of the features of `raspistill`.
+* _libcamera-vid_ A video capture application.
+* _libcamera-raw_ A basic application for capturing raw (unprocessed Bayer) frames directly from the sensor.
+
+Raspberry Pi's `libcamera-apps` are not only command line applications that make it easy to capture images and video from the camera, they are also examples of how users can create their own libcamera-based applications with custom functionality to suit their own requirements. The source code for the `libcamera-apps` is freely available under a BSD 2-Clause licence at https://github.com/raspberrypi/libcamera-apps[].
+
+==== More about `libcamera`
+
+`libcamera` is an open source Linux community project. More information is available at the https://libcamera.org[`libcamera` website].
+
+The `libcamera` source code can be found and checked out https://git.linuxtv.org/libcamera.git/[here].
```

Comment:
I think @aallan doesn't like the use of "here" as link-text.

Reply:
Nope. Not allowed (and I'm going through the existing documentation trying to remove 'em all). Documentation should be written as if it were going to be printed out and the links didn't work. Then add the links in afterwards.